projects/crosvm/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "http://crosvm.dev"
+language: rust
+primary_contact: "denniskempin@google.com"
+auto_ccs:
+  - "crosvm-core@google.com"
```

Comment:
note these folks won't be able to access bugs directly, they will need to add their emails explicitly for that.